scripts/qa_crowbarsetup.sh

```diff
@@ -4722,6 +4722,16 @@ function onadmin_crowbarrestore
     fi
 }
 
+
+function onadmin_allow_vendor_change_at_nodes
+{
+    if [[ $cloudsource = develcloud6 ]] ; then
```

Comment:
Shouldn't we enable this also when ```upgrade_cloudsource``` comes from develcloud-something? (This will allow testing GM6 to develcloud7 upgrades, which makes a lot of sense).

I'd suggest to move the cloudsource check to ```mkcloud``` (IIRC upgrade_cloudsource might not be available in qa_crowbarsetup) and remove the condition here.